lib/bitgo_client/v2.rb

```diff
@@ -25,23 +26,23 @@ def client
       BitgoClient::Client.new(access_token)
     end
 
-    def create_address(wallet_id, coin_code: :tbtc, logger: nil, proxy: nil)
+    def create_address(wallet_id, coin_code: :tbtc, logger: nil)
       client.request("#{base_path}/#{coin_code}/wallet/#{wallet_id}/address", method: :post, logger: logger, proxy: proxy)
```

Comment:
Got some more :) On the same vein, you can just pass proxy into `Client` like `access_token` is, so that you don't even need to pass it into `client.request`.